deployment/nginx.coffee

```diff
@@ -365,6 +365,17 @@ module.exports.create = (KONFIG, environment)->
         resolver 8.8.8.8;
       }
 
+      # redirect /s3/:bucket/:key to :bucket.s3.amazonaws.com/:key
+      location ~^/s3/([^/]*)/(.*)$ {
```

Comment:
@cihangir on a second thought, wouldn't it be better to redirect instead of revproxy?